lib/settings.js

```diff
@@ -0,0 +1,36 @@
+var fs = require('fs');
+var AppDirectory = require('appdirectory');
+var dirs = new AppDirectory('sql-seer');
+var configFile = dirs.userConfig() + "/settings.json";
```

Comment:
I created another RP https://github.com/EvolveLabs/electron-updater/pull/29
since I don't know how to correct the bin/cli file. Please help me on that.